src/app/donation-start/updateDonationFromForm.ts

```diff
@@ -0,0 +1,65 @@
+import {StepperSelectionEvent} from "@angular/cdk/stepper";
+import {AnalyticsService} from "../analytics.service";
+import {FormGroup} from "@angular/forms";
+import {DonationService} from "../donation.service";
+import {Donation} from "../donation.model";
+import {Campaign} from "../campaign.model";
+import {sanitiseCurrency} from "./sanitiseCurrency";
+
+
+export function updateDonationFromForm
```

Comment:
There are actually no tests that cover this function - returning void at the first line doesn't make anything fail. So no tests to move out of the component spec file.